src/main/resources/META-INF/groovy/org.codehaus.groovy.runtime.ExtensionModule

```diff
@@ -0,0 +1,3 @@
+moduleName=JfrUnit extensions
+moduleVersion=0.1
```

Comment:
Should this be the same version as the artifact version itself? If so, I suggest we automatically add this via Maven resource filtering.

Reply:
Yes, makes sense and seems to be safer. Changed. Please review as this is me using the resource filtering for the first time.

Reply:
LGTM.